src/main/java/com/gregtechceu/gtceu/api/blockentity/MetaMachineBlockEntity.java

```diff
@@ -317,4 +345,20 @@ public AABB getRenderBoundingBox() {
         }
         return new AABB(worldPosition.offset(-1, 0, -1), worldPosition.offset(2, 2, 2));
     }
+
+    @Override
+    protected void saveAdditional(CompoundTag tag) {
+        super.saveAdditional(tag);
```

Comment:
OwnerType should probably also be serialized in some way (class name ideally), so that owners of the wrong type aren't being loaded when one of the teams mods is removed.